app-cat-clicker/Controllers/cat1controller.js

```diff
@@ -0,0 +1,15 @@
+//h-w-1 section 1
+(function(module) {
+  module.controller("cat1Controller",cat1Controller);
+
+})(angular.module("catclick"));
+
+
+function cat1Controller($scope) {
```

Comment:
function cat1Controller should be placed inside the IFFE block, As it is not in IFFE block it will in the globalScope.